Magiczne-Schody/IChainOfResponsibility.h

```diff
@@ -0,0 +1,18 @@
+/*******************************************************
+* Copyright (C) 2018 Mateusz Urbanek <mateuszurbanek@gmail.com>
+*
+* This file is part of project Magiczne Schody.
+*
+* Project Magiczne Schody can not be copied and/or distributed without the express
+* permission of Malgorzata Gora <gora.malgorzata.b@gmail.com>
+*******************************************************/
+#pragma once
+
+class IChainOfResponsibility
+{
+public:
+	// pure virtual functions providing interface framework.
+	virtual void executeCommand(char eCommand) = 0;
```

Comment:
Tu kolejna prośba na zmianę nazwy z executeCommand na processRequest

We wzorcach projektowych bardzo ważne jest aby posługiwać się nazwami ogólnie przyjętymi dla danego wzorca bo to upraszcza czytanie kodu przez innych i jego łatwiejsze zrozumienie.